assets/js/blocks/product-gallery/inner-blocks/product-gallery-pager/block.json

```diff
@@ -8,6 +8,6 @@
     "category": "woocommerce",
     "keywords": [ "WooCommerce" ],
     "textdomain": "woo-gutenberg-products-block",
-	"ancestor": [ "woocommerce/product-gallery" ],
+	"ancestor": [ "woocommerce/product-gallery-large-image" ],
```

Comment:
This doesn't seem to be correct, as now the Large Image is Pager's sibling.

<img width="414" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/e59218bc-e91a-486b-b23e-1e7b1d4c9697">

Reply:
Nice catch! The ancestor should be the Product Gallery block, I just updated this line of code